scripts/actions/mobskills/aura_of_persistence.lua

```diff
@@ -6,11 +6,17 @@
 local mobskillObject = {}
 
 mobskillObject.onMobSkillCheck = function(target, mob, skill)
+    if mob:hasStatusEffect(xi.effect.DEFENSE_BOOST) then
+        return 1
+    end
+
     return 0
 end
 
 mobskillObject.onMobWeaponSkill = function(target, mob, skill)
-    skill:setMsg(xi.mobskills.mobBuffMove(mob, xi.effect.DEFENSE_BOOST, 17.5, 0, 300))
+    local duration = xi.mobskills.calculateDuration(skill:getTP(), 30, 60)
+    local power = 1 + 52 / 256
```

Comment:
1+52/256 = 1.203125
DEFP is an integer that adds by percent, so this is approximated by 

```suggestion
    local power = 20
```

Reply:
Got that adjusted to 20